packages/frontend/src/components/navigation/NavigationWrapper.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC, useCallback } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { switchAccount } from '../../redux/actions/account';
+import { selectAccountSlice } from '../../redux/slices/account';
+import {
+    selectFlowLimitationMainMenu,
+    selectFlowLimitationSubMenu,
+} from '../../redux/slices/flowLimitation';
+import Navigation from './Navigation';
+
+const NavigationWrapper: FC = () => {
+    const dispatch = useDispatch();
+    const account = useSelector(selectAccountSlice);
+    const flowLimitationMainMenu = useSelector(selectFlowLimitationMainMenu);
+    const flowLimitationSubMenu = useSelector(selectFlowLimitationSubMenu);
+
+    const selectAccount = useCallback((accountId) => {
+        dispatch(switchAccount({ accountId }));
```

Comment:
For now it can't be.
https://github.com/mynearwallet/near-wallet/blob/ea762d1c480843a45c2fcfce6f810d0a07f4d39b/packages/frontend/src/redux/actions/account.js#L555

We can change it.